### PR TITLE
fix(data-source-currency): set default dataSource currency

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [x] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA
image

### CODE REVIEW

File: apps/web/src/services/cost-explorer/CostExplorerLNB.vue [hunk 1]
```diff
@@ -163,9 +163,13 @@ const filterFavoriteItems = (menuItems: LNBItem[] = []): LNBItem[] => {
     return menuItems.filter((menu) => (menu.id && state.favoriteItemMap[menu.id]) || menu.type !== MENU_ITEM_TYPE.ITEM);
 };
 
-const getCurrenctCurrency = (dataSourceKey: string): string => dataSourceState.dataSourceMap[dataSourceKey].data.plugin_info.metadata.currency;
-
-const getCurrencySymbol = (currency: string): string => CURRENCY_SYMBOL[currency];
+const getCurrentCurrencySet = (dataSourceKey: string): string => {
+    const defaultCurrencySet = `${CURRENCY_SYMBOL.USD}${CURRENCY.USD}`;
+    const currentCurrency = dataSourceState.dataSourceMap[dataSourceKey].data.plugin_info.metadata.currency;
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/CostExplorerLNB.vue]: question: no need to set optional mark in here?

**Review by yuda110 — APPROVED** [review]
LGTM!! I left a question

File: apps/web/src/services/cost-explorer/CostExplorerLNB.vue (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/services/cost-explorer/CostExplorerLNB.vue]: I think it is necessary. Thanks!

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Pull request merged**